### ISSUE TITLE
CC a11y audit only - file input component

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#5547]
Audit the file input component for WCAG compliance and critical checklist checks using three manual accessibility testing methods.

[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)

This ticket will be considered done once the following content has been written and associated WCAG checks are in place for the file input component:

WCAG criteria related to the [file input ](https://designsystem.digital.gov/components/file-input/)component

How to perform manual testing (critical checklists) and related user-focused questions for downstream teams to test.
a. Zoom mag to 200%
b. Keyboard
c. JAWS
### A11Y
- [x] Establish the success criteria we need to check against 
- [x] Perform each check 
- [x] Review with other a11y team member
- [x] Reassess (as needed)
- [x] Submit any failures as bug tickets

### Content and Dev

- [ ] https://github.com/uswds/uswds-site/issues/2431

## References
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
[List of all components](https://docs.google.com/spreadsheets/d/1ASzZ80oI6WVWAifqPO6gYAqrsRaiZs_qGTOStuQPyKs/edit?usp=sharing)

**Comment by thisisdano** [comment]
Done
